Cargo.lock

```diff
@@ -7192,7 +7192,7 @@ dependencies = [
 [[package]]
 name = "trustify-ui"
 version = "0.1.0"
-source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#b2d81aec3218a900bbb73bcdb69d483e96cbc48f"
+source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#b467f98f72dcd12ec2a82c34da48388a5e050618"
```

Comment:
```suggestion
source = "git+https://github.com/trustification/trustify-ui.git?tag=static-main#857f4bfbc59c668fd1379360d7378c18df3f7f9a"
```

Reply:
I have added a commit to the ui in order to reflect the latest changes about /api/v1/package -> /api/v1/purl so the UI works as it used to work until this Monday demo.